Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -118,6 +122,17 @@ public struct CustomerCenterConfigData {
                     return "Support Request"
                 case .dismiss:
                     return "Dismiss"
+                case .updateWarningTitle:
+                    return "Update the app to fix your problem"
+                case .updateWarningDescription:
+                    return """
+                    A new update is available. Downloading the latest version of the app can help you solve your \
+                    problem.
```

Comment:
I'd consider this slight change in wording.
```suggestion
                    A new update is available. Downloading the latest version of the app may help solve the \
                    problem.
```